fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -167,6 +168,7 @@
     public static final String RUNTIME_JOIN_FILTER_PUSH_DOWN_LIMIT = "runtime_join_filter_push_down_limit";
     public static final String ENABLE_GLOBAL_RUNTIME_FILTER = "enable_global_runtime_filter";
     public static final String ENABLE_COLUMN_EXPR_PREDICATE = "enable_column_expr_predicate";
+    public static final String CTE_MEM_LIMIT = "cte_mem_limit";
```

Comment:
Do we really need a  session variables?  we should add a new session variables if necessary.